source/help/settings/account-settings.rst

```diff
@@ -196,10 +196,16 @@ Website Link Previews
 
 When available, the first web link in a message will show a preview of the website content below the message. This `setting must be enabled by your System Admin <https://docs.mattermost.com/administration/config-settings.html#link-previews>`__.
 
-Link Previews
-~~~~~~~~~~~~~
+Default Appearance of Image Previews
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When messages in Mattermost include images, an image preview can display directly below the message for image attachments, image link previews, and `in-line images <https://docs.mattermost.com/help/messaging/formatting-text.html#in-line-images.html>`__ over 100px in height.
+
+Enable this preference to show image previews. Disable this preference to hide link previews in Mattermost.
```

Comment:
enabling/disabling just sets the default expanded/collapsed states, so technically not "hiding"